apps/E2E/src/common/BasePage.ts

```diff
@@ -238,6 +242,49 @@ export abstract class BasePage {
     await FocusButton.addValue(scrollDownKeys);
   }
 
+  /* Scrolls to the specified or primary UI test element until it is displayed. */
+  async mobileScrollToTestElement(componentIdentifier?: string): Promise<void> {
+    const componentToScrollTo = componentIdentifier ?? this._primaryComponentName;
+    if (await (await By(componentToScrollTo)).isDisplayed()) {
+      return;
+    }
+
+    const errorMsg = 'Could not scroll to the ' + componentToScrollTo + '. Please see Pipeline artifacts for more debugging information.';
+
+    switch (this.platform) {
+      case MobilePlatform.iOS: {
```

Comment:
Is this actually needed for iOS? iOS hasn't been using this before, and it's been passing successfully.